Expecto/Expecto.fs

```diff
@@ -330,7 +330,7 @@ module Impl =
       >> setField "failed" failed
       >> setField "errored" errored)
 
-
+  let semaphore = obj ()
```

Comment:
So you'd like to make the change in the upstream Logary.Facade before this PR is ready to be merged?